spec/feature/posts_show_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe 'Post show page', type: :feature do
-  let!(:user) { User.create(name: 'Sarkodie', photo: '/path/to/photo1.jpg', post_counter: 10) }
+  let!(:user) { User.create(name: 'Sarkodie', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', posts_counter: 10) }
   let!(:post) do
     Post.create(
```

Comment:
- [ ]  At the moment you are missing the test that asserts that you can see who wrote the post. Kindly add it.